62.CAD/fragment_shader.hlsl

```diff
@@ -40,14 +115,33 @@ float4 main(PSInput input) : SV_TARGET
     }
     else if (objType == ObjectType::QUAD_BEZIER)
     {
-        const float2 a = input.getBezierP0();
-        const float2 b = input.getBezierP1();
-        const float2 c = input.getBezierP2();
+        nbl::hlsl::shapes::Quadratic<float> quadratic = input.getQuadratic();
+        QuadBezierAnalyticArcLengthCalculator<float> preCompValues_calculator = input.getPrecomputedArcLenData();
+        nbl::hlsl::shapes::Quadratic<float>::ArcLengthPrecomputedValues preCompValues;
+        preCompValues.lenA2 = preCompValues_calculator.lenA2;
+        preCompValues.AdotB = preCompValues_calculator.AdotB;
+        preCompValues.a = preCompValues_calculator.a;
+        preCompValues.b = preCompValues_calculator.b;
+        preCompValues.c = preCompValues_calculator.c;
+        preCompValues.b_over_4a = preCompValues_calculator.b_over_4a;
+        
+        const uint32_t styleIdx = mainObjects[currentMainObjectIdx].styleIdx;
         const float lineThickness = input.getLineThickness();
+        float distance;
+        
+        if (lineStyles[styleIdx].stipplePatternSize == 0u)
```

Comment:
make a bool function in lineStyle struct named `hasStipples` or something

Reply:
done